pkg/util/testing/error_matchers.go

```diff
@@ -76,26 +76,47 @@ func (matcher *isForbiddenErrorMatch) NegatedFailureMessage(actual interface{})
 	return format.Message(actual, "not to be Forbidden error")
 }
 
-func BeInvalidError() types.GomegaMatcher {
-	return &isInvalidErrorMatch{}
+type errorMatcher int
```

Comment:
You could also reimplement `BeForbiddenError` as `return BeAPIError(ForbiddenError)`. Similarly for `BeNotFoundError`